frontend/src/components/Comments/SingleComment/Comment.js

```diff
@@ -17,7 +17,7 @@ export const UserComment = ({ comment }) => {
     minute: '2-digit',
   });
   return (
-    <CommentContainer data-testid="userComment">
+    <CommentContainer data-testid="comment">
```

Comment:
```suggestion
    <Container>
```
Używasz gdzieś tego id?

Reply:
Id wyrzucone. CommentContainer nazwane zostało świadomie: 
w miejscu w którym go wykorzystuję tworzę obiekt o nazwie Comment. 
![image](https://user-images.githubusercontent.com/22809303/150686707-dcc37bcd-bd30-42ad-8af1-0fbd46d21845.png)

Reply:
Ale czy ten container nie wie że jest w komponencie Comment?
A czy pojawia się gdzieś w tym komponencie jeszcze container?

Reply:
W sensie, jeżeli zamiast CommentContainer zastosuję nazwę Comment to mam zarówno styl, jak i komponent o tej samej nazwie.